engine/src/main/java/io/camunda/zeebe/engine/state/migration/DbMigrationState.java

```diff
@@ -387,6 +422,14 @@ public void migrateProcessDefinitionVersions() {
 
   @Override
   public void migrateProcessStateForMultiTenancy() {
+    deprecatedProcessCacheColumnFamily.forEach(
+        (key, value) -> {
+          value.setTenantId(TenantOwned.DEFAULT_TENANT_IDENTIFIER);
+          tenantIdKey.wrapString(TenantOwned.DEFAULT_TENANT_IDENTIFIER);
```

Comment:
❓ WDYT about wrapping this in the constructor. In these migrations it should always get the default value anyway, no need to wrap it each time.

Only downside I can see is that we may get a later migration that uses it as well, but even then I don't see an issue as long as we don't run those migration earlier, which we shouldn't do anyway.

Reply:
Resolved in a later commit